Source/Core/Core/IOS/IOS.h

```diff
@@ -84,7 +84,10 @@ enum ProcessId : u32
 class Kernel
 {
 public:
+  // Used to initialise an IOS instance that is tied to PPC emulation.
   explicit Kernel(u64 ios_title_id);
+  // For internal uses. This only initialises core modules (ES, FS) and skips everything else.
+  Kernel();
```

Comment:
Not too thrilled about this ctor (or rather, about it being public), but I'm not sure how to work around this. You could make a static (or perhaps even non-static?) method somewhere inside `IOS` or `Kernel` that internally does all those things (so we don't have to make the ctor publicly available for everyone to abuse and break stuff). But that feels just as dirty as this one...